src/material-experimental/mdc-tabs/tab-group.spec.ts

```diff
@@ -881,3 +913,18 @@ class TabGroupWithIsActiveBinding {
   `,
 })
 class TabsWithCustomAnimationDuration {}
+
+
+@Component({
+  template: `
+    <mat-tab-group>
+      <ng-container [ngSwitch]="true">
+        <mat-tab label="One">Tab one content</mat-tab>
+        <mat-tab label="Two">Tab two content</mat-tab>
+      </ng-container>
+    </mat-tab-group>
+  `,
+})
+class TabGroupWithIndirectDescendantTabs {
```

Comment:
This is all duplicated from what's in `material/`. Do we have a reason for choosing this approach?
Was it too much work or is there a blocker to trying to export something from `material/` tests that could be re-used in the `material-experimental/mdc-x` tests?